share/resources/completion.json

```diff
@@ -0,0 +1,85 @@
+{
+    "keywords": {
+        "GO" : {
+            "sub_keywords": [
+                "UPTO", "STEPS", "FROM", "OVER", "AS", "WHERE", "YIELD", "REVERSELY"
+            ]
+        },
+        "INSERT" : {
+            "sub_keywords": [
+                "VERTEX", "EDGE", "VALUES", "NO", "OVERWRITE"
+            ]
+        },
+        "SHOW" : {
+            "sub_keywords": [
+                "HOSTS", "TAGS", "EDGES", "SPACES", "USERS", "ROLES", "USER"
+            ]
+        },
+        "DESCRIBE" : {
```

Comment:
`DESC` is miss ?

Reply:
It was not available when this PR initiated.

Let me add it in.